examples/backtest.ipynb

```diff
@@ -23,12 +23,14 @@
     "\n",
```

Comment:
<p>To visualize the train part, you can specify the <code>history_len</code> parameter. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>